site/releases/2024-jan-22/highlights.qmd

```diff
@@ -0,0 +1,218 @@
+---
+title: "January 22, 2024"
+keywords: "release notes, ai risk, model risk management, ValidMind"
+---
+
+## Release highlights
+
+This release includes numerous improvements to the ValidMind Developer Framework, including new features for model and dataset initialization, easier testing, support for additional inputs and the Azure OpenAI API, updated notebooks, bug fixes, deprecations, and much more.
+
+### ValidMind Developer Framework (v1.25.3)
+
+<!---[SC-2628] `init_model` improvements  by @AnilSorathiya in [#304](https://github.com/validmind/developer-framework/pull/304)--->
+
+#### Improvements to `init_model`
+
+When initializing a model, you can now pass a dataset with pre-computed model predictions to improve performance. By default, if no prediction column is specified when calling `init_model`, the ValidMind Developer Framework will compute the model predictions as before.
+
+To illustrate how passing a dataset that includes a prediction column can help, consider the following example without a prediction column: 
+
+```python
+vm_model = vm.init_model(
+    model,
+    train_ds=vm_train_ds,
+    test_ds=vm_test_ds,
+)
+```
+
+Internally, this example invokes the `predict()` method of the model for the training and test datasets when the model is initialized. This approach can be problematic with large datasets: `init_model` can simply take too long to compute.
+
+You can now avoid this issue by providing a dataset with a column containing pre-computed predictions, similar to the following example. If `init_model` detects this column, it will not generate model predictions at runtime. 
+
+|x1|x2|...|target_column|prediction_column|
+|-|-|-|-|-|
+|0.1|0.2|...|0|0|
+|0.2|0.4|1...|1|1|
+
+Usage example with a prediction column:
+
+```python
+vm.init_dataset(
+     dataset=df,
+     feature_columns=[...],
+     target_column= ...,
+     extra_columns={
+         prediction_column: 'NAME-OF-PREDICTION-COLUMN',
+    },
+)
+```
+
+<!---[SC-2438] Support for adding feature columns parameter to init dataset by @juanmleng in [#279](https://github.com/validmind/developer-framework/pull/279)--->
+#### Improvements to `init dataset`
```

Comment:
I believe it should be "init_dataset" (not "init dataset")

Reply:
Good catch! Fixed.